Drivers/MotorChannel/Inc/ZP_D_MotorChannel.hpp

```diff
@@ -0,0 +1,13 @@
+#ifndef ZP_D_MOTOR_CHANNEL_HPP_
+#define ZP_D_MOTOR_CHANNEL_HPP_
+
+#include <cstdint>
+
+class MotorChannel {
+    public:
+        virtual void set(uint8_t percent) = 0;
+    protected:
+        bool isSetup = false;
```

Comment:
This `isSetup` variable doesn't seem to have any utility the way the driver is currently structured, and should probably just be removed.